vue-frontend/src/components/home/HowDoWeDiffer.vue

```diff
@@ -0,0 +1,313 @@
+            >
+          </div>
+          <div class="text-center normal-text text-black">
+            <object type="image/svg+xml" :data="seoIcon" class="pb-4"></object>
+            <b>
+              SEO and <br />
+              Marketing</b
+            >
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script type="module">
+import cloudIcon from "@/assets/images/icon/cloud.svg";
+import customIcon from "@/assets/images/icon/custom.svg";
+import desktopIcon from "@/assets/images/icon/desktop.svg";
+import mobileIcon from "@/assets/images/icon/mobile.svg";
+import qaIcon from "@/assets/images/icon/qa.svg";
+import scrollArrowToUpIcon from "@/assets/images/icon/scroll-arrow-to-up.svg";
+import seoIcon from "@/assets/images/icon/seo.svg";
+import uiuxIcon from "@/assets/images/icon/uiux.svg";
+import watchIcon from "@/assets/images/icon/watch.svg";
+import webIcon from "@/assets/images/icon/web.svg";
+
+export default {
+  data() {
+    return {
+      cloudIcon: cloudIcon,
+      customIcon: customIcon,
+      desktopIcon: desktopIcon,
+      mobileIcon: mobileIcon,
+      qaIcon: qaIcon,
+      scrollArrowToUpIcon: scrollArrowToUpIcon,
+      seoIcon: seoIcon,
+      uiuxIcon: uiuxIcon,
+      watchIcon: watchIcon,
+      webIcon: webIcon,
+    };
+  },
+  mounted() {
+    if (window.innerWidth >= 600 && window.innerWidth <= 992) {
+      this.$refs.flexGridTwo.prepend(this.$refs.uiUxDiv);
+    } else {
+      this.$refs.flexGridOne.append(this.$refs.uiUxDiv);
+    }
+  },
+};
+</script>
+
+<style scoped>
+.m-lr-10 {
+  margin: 0 10%;
+}
+
+.light-orange-bg {
+  background: rgba(255, 148, 114, 0.05);
+  z-index: 100;
+  margin-top: 0;
+}
+
+.light-orange-bg > .m-lr-10 > .description {
+  padding-top: 80px;
+  margin: auto;
+}
+
+.steve-box {
+  margin: 2rem 0.5rem;
+  line-height: 2;
+  padding: 1.5rem;
+  border-radius: 0.9rem;
+  text-align: left;
+  border: 1px solid transparent !important;
+  position: relative;
+}
+
+.gradient-border {
+  background: -webkit-linear-gradient(
+        rgba(255, 250, 248, 255),
+        rgba(255, 250, 248, 255)
+      )
+      padding-box,
+    linear-gradient(to bottom, #ff9472, #f2709c) border-box;
+  border: 2px solid transparent;
+  border-radius: 15px;
+  display: inline-block;
+}
+
+.orange-gradient-text {
+  background: linear-gradient(180deg, #ff9472 0%, #f2709c 100%);
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+}
+
+.flex-grid {
+  display: flex;
+  flex-direction: column;
+  padding-bottom: 7%;
+}
+
+.flex-grid-1,
+.flex-grid-2 {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: unset;
+}
+
+.flex-grid-1 > div,
+.flex-grid-2 > div {
+  padding: 1.25rem;
+  width: 50%;
+  display: flex;
+  flex-direction: column;
+}
+
+.flex-grid-1 > div > object,
+.flex-grid-2 > div > object {
+  height: 7.5rem;
+}
+
+.flex-grid-1 > div > b,
+.flex-grid-2 > div > b {
+  font-size: 1.125rem;
+}
+
+@media (min-width: 481px) {
```

Comment:
I think we are doing mobile-first CSS. There should be no need to separate CSS for mobile widths.